src/psyclone/tests/psyGen_test.py

```diff
@@ -676,7 +676,7 @@ def test_codedkern_class_view(capsys):
 
 
```

Comment:
There are a few tests with the docstring:
```
src/psyclone/tests/psyGen_test.py:2915:    ''' Check the view and colored_text methods of the Code Block class.'''
src/psyclone/tests/psyGen_test.py:3089:    ''' Check the view and colored_text methods of the IfBlock class.'''
src/psyclone/tests/psyGen_test.py:3178:    ''' Check the view and colored_text methods of the Assignment class.'''
src/psyclone/tests/psyGen_test.py:3225:    ''' Check the view and colored_text methods of the Reference class.'''
src/psyclone/tests/psyGen_test.py:3308:    ''' Check the view and colored_text methods of the Array class.'''
src/psyclone/tests/psyGen_test.py:3340:    ''' Check the view and colored_text methods of the Literal class.'''
src/psyclone/tests/psyGen_test.py:3379:    ''' Check the view and colored_text methods of the Binary Operation
src/psyclone/tests/psyGen_test.py:3431:    ''' Check the view and colored_text methods of the UnaryOperation
src/psyclone/tests/psyGen_test.py:3456:    ''' Check the view and colored_text methods of the Nary Operation
src/psyclone/tests/psyGen_test.py:3488:    ''' Check the view and colored_text methods of the Return class.'''
src/psyclone/tests/psyGen_test.py:3538:    '''Check the view and colored_text methods of the Container class.'''
```
which I think the name of the tests and the methods in the docstring should be changed to node_str and colored_name. The body of the functions is still essentially the same but comparing with `node_str(True)` instead of `view()` and `capsys.readouterr()`

Reply:
Good point. I've done the majority of them. There were a couple where it still made sense to test view() so I've left those.